ui/helpers/utils/util.js

```diff
@@ -524,3 +525,20 @@ export function coinTypeToProtocolName(coinType) {
 export function isNullish(value) {
   return value === null || value === undefined;
 }
+
+/**
+ * The method escape RTL character in string
+ *
+ * @param {string} str
+ * @returns {(string|*)} escaped string or original param value
+ */
+export const sanitizeString = (str) => {
```

Comment:
I suppose the `str` param could be type any (*) now and maybe we want to rename the parameter to `value`. 

That, or we could allow this function to convert the value by renaming to function, e.g. `stringifyAndSanitize`, or adding an optional param, `stringify = true`

Reply:
Good point @digiwand : I renamed arg to `value`.